master/static/srv/k8_init_container_entrypoint.sh

```diff
@@ -13,6 +13,9 @@ do
     SRC=$2/$IDX.tar.gz
     DST=$3/$IDX
     mkdir $DST
-    # TODO(Brad): init container runs as root to use --same-owner, could be a problem.
-    tar --same-owner -xvf $SRC -C $DST
+
+    if [ -f $SRC ]
```

Comment:
Somewhat lazy on my part but the idea in the current code is volumes references directories indexed ```0-8``` to mount files in the correct place

Rather than changing the code to smartly understand which files won't be copied through the tar -> volume system (because they will be added as root in the config map)

Think the refactor of handling root files in one place will make this go away